telegram/_inline/inlinekeyboardbutton.py

```diff
@@ -145,6 +145,12 @@ class InlineKeyboardButton(TelegramObject):
             be launched when the user presses the button.
         pay (:obj:`bool`): Optional. Specify :obj:`True`, to send a Pay button.
 
+    .. seealso:: `inlinekeyboardbutton.py <https://github.com/python-telegram-bot/
+        python-telegram-bot/blob/master/telegram/_inline/inlinekeyboardbutton.py>`_,
+        `inlinekeyboard2.py <https://github.com/python-telegram-bot/
+        python-telegram-bot/blob/master/examples/inlinekeyboard2.py>`_,
+        `InlineKeyboard Example <https://github.com/python-telegram-bot/
+        python-telegram-bot/wiki/InlineKeyboard-Example>`_
```

Comment:
```suggestion
        `Inline Keyboard Example 1<examples.inlinekeyboard.html>`_,
        `Inline Keyboard Example 2<examples.inlinekeyboard2.html>`_
```
since we host examples on docs now, we should refer to there. Apply this to all other places.

Note to self/devs: ```:doc:`examples.inlinekeyboard` ``` does not work in `.. seealso::`